### ISSUE TITLE
lightspark for android ???

### ISSUE THREAD

**Issue by ghost** [lightspark/lightspark#209]
lightspark , when support for the android

**Comment by paultutao** [comment]
Have any transfer lightspark to Android 4.4 platform?

**Comment by abasgames** [comment]
.

**Comment by MrWildanMD** [comment]
can i just write a native android app that loads lib lightspark via JNI?

**Comment by MrWildanMD** [comment]
> > can i just write a native android app that loads lib lightspark via JNI?
> 
> We have a basic gui so I guess, would just have to use GameActivity and add a android_main to lightspark I guess

Yes i think so, but what i want to ask, did the lightspark requirement/lib is satisfied in android...or i need to compile also the requirement/lib/dependency in supported android way? like audio for example, thanks

**Comment by dbluelle** [comment]
You have to check if all the lightspark dependencies (SDL2, ffmpeg, cairo, pango, ...)  are available on Android.
I doubt that those are included in the NDK, but maybe there are binaries provided by the developers for some of them.
For those that aren't you have to compile them yourself.
